src/jsp/com/cubrid/jsp/jdbc/CUBRIDServerSideConnection.java

```diff
@@ -92,26 +93,39 @@ public SUConnection getSUConnection() {
     }
 
     protected void requestDBParameter() throws IOException, SQLException {
-        DBParameterInfo info = suConn.getDBParameter();
-        switch (info.tran_isolation) {
-            case CUBRIDIsolationLevel.TRAN_READ_COMMITTED:
-                transactionIsolation = TRANSACTION_READ_COMMITTED;
-                break;
-
-            case CUBRIDIsolationLevel.TRAN_REPEATABLE_READ:
-                transactionIsolation = TRANSACTION_REPEATABLE_READ;
-                break;
-
-            case CUBRIDIsolationLevel.TRAN_SERIALIZABLE:
-                transactionIsolation = TRANSACTION_SERIALIZABLE;
-                break;
-
-            default:
-                transactionIsolation = TRANSACTION_NONE;
-                break;
+        DBParameterInfo info = getSUConnection().getDBParameter();
+        if (transactionIsolation == TRANSACTION_NONE) {
```

Comment:
Why is this if statement condition necessary?

Reply:
It is not necessary, I'm going to remove the if statement